rules/share-your-developer-secrets-securely/rule.md

```diff
@@ -0,0 +1,112 @@
+---
+type: rule
+archivedreason:
+title: Do you share your developer secrets securely?
+guid: 6165deaa-a3b0-40b1-8659-482a34faf7c4
+uri: share-your-developer-secrets-securely
+created: 2023-08-14T00:00:00.0000000Z
+authors:
+  - title: Gordon Beeming
+    url: https://ssw.com.au/people/gordon-beeming
+related: 
+  - store-your-secrets-securely
+redirects: []
+---
+
+Most systems will have variables that need to be stored securely; OpenId shared secret keys, connection strings, and API tokens to name a few.
+
+These secrets **must not** be stored in source control. It is insecure and means they are sitting out in the open, wherever code has been downloaded, for anyone to see.
```

Comment:
I prefer "not secure" to "insecure". Just a preference though.

Reply:
I like not secure, sounds more absolute for some reason